cms/migrations/0030_populate_program_courses.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-05-08 20:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def populate_program_courses(apps, schema_editor):
+    ProgramCourse = apps.get_model('cms', 'ProgramCourse')
+    ProgramPage = apps.get_model('cms', 'ProgramPage')
+
+    program_pages = ProgramPage.objects.all()
+
+    for program_page in program_pages:
+        courses = program_page.program.course_set.all()
+        for course in courses:
+            ProgramCourse.objects.create(
+                course=course,
+                title=course.title,
+                description=course.description,
+                program_page=program_page
+            )
+
+
+def reverse_populate_program_courses(apps, schema_editor):
+    ProgramCourse = apps.get_model('cms', 'ProgramCourse')
```

Comment:
We might actually want to do a `RunPython.noop` for the reverse migration here, my thinking is if we reverted this migration we would end up in a state with no `ProgramCourse`s.